test/common/autoscaling.go

```diff
@@ -0,0 +1,98 @@
+package common
+
+import (
+	"context"
+	rhmiv1alpha1 "github.com/integr8ly/integreatly-operator/apis/v1alpha1"
+	"github.com/integr8ly/integreatly-operator/pkg/addon"
+	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"time"
+)
+
+const (
+	hpaQuotaParam = "500"
+	hpaQuotaName  = "50 Million"
+)
+
+func TestAutoscaling(t TestingTB, ctx *TestingContext) {
+	quotaConfig, err := getQuotaConfig(t, ctx.Client)
+	if err != nil {
+		t.Fatalf("Error retrieving Quota: %v", err)
+	}
+
+	installation, err := GetRHMI(ctx.Client, true)
+	if err != nil {
+		t.Fatal("couldn't get RHMI cr for quota test")
+	}
+	if installation == nil {
+		t.Fatalf("Got invalid rhmi CR: %v", installation)
+	}
+	initialHPAValue := installation.Spec.AutoscalingEnabled
+
+	// wait if stage no complete
+	startTime := time.Now()
+	endTime := startTime.Add(time.Minute * time.Duration(timeoutWaitingQuotachange))
+
+	for startTime.Before(endTime) {
```

Comment:
This is not required. The check if installation is completed is already done here:
https://github.com/integr8ly/integreatly-operator/blob/master/test/functional/integreatly_test.go#L24 when it comes to functionals and here: https://github.com/integr8ly/integreatly-operator/blob/master/test/e2e/integretly_test.go#L24 when it comes to prow e2e